.ci/docker/ci_commit_pins/triton.txt

```diff
@@ -1 +1 @@
-96316ce50fade7e209553aba4898cd9b82aab83b
+6da9e66008b58a7b8553f96c69021cca0d0028f0
```

Comment:
This commit needs to picked from - https://github.com/triton-lang/triton/commits/release/3.3.x/
Tip is fine - 819e9c8c29ad2ae96cbd93a1d3b8a3a0f4c8f09c